src/components/PhoneInput/__tests__/phoneinput.spec.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import PhoneInput from '../';
+
+const Wrapper = () => {
+    const [value, setValue] = React.useState('');
+    return (
+        <PhoneInput
+            label="Phone Number"
+            value={value}
+            onChange={setValue}
+            placeholder="Enter your phone number"
+        />
+    );
+};
+
+describe('<PhoneInput />', () => {
+    it('Should accept only numbers.', () => {
+        const wrapper = mount(<Wrapper />);
```

Comment:
I think here you can use the PhoneInput directly and instead check for the onChange event like this:
```
const onChangeMockFn = jest.fn();
const component = mount(< PhoneInput onChange={onChangeMockFn} />);
// simulate
expect(onChangeMockFn).toHaveBeenCalledWith(...);
```

Reply:
In the way you suggest, how do I verify that the input only contains numbers?

Reply:
@devrchancay instead of doing:
```expect(wrapper.find('input[type="tel"]').prop('value')).toBe('0987256983');```
you can do:
```
expect(onChangeMockFn).toHaveBeenCalledWith(expect.objectContaining({
    phone: '0987256983',
}));
```

Reply:
Oh thanks!